cmd/minikube/cmd/delete.go

```diff
@@ -145,6 +146,41 @@ func deleteContainersAndVolumes(ctx context.Context, ociBin string) {
 	}
 }
 
+// kicbaseImage fetches images based on container label
+func kicbaseImages(ctx context.Context, ociBin string) ([]string, error) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return nil, nil
+	}
+	delLabel := fmt.Sprintf("%s=%s", oci.CreatedByLabelKey, "true")
+	names, err := oci.ListImagesByContainerLabel(ctx, ociBin, delLabel)
```

Comment:
One last thing, we might as well do an inline return here since that's all we're effectively doing anyway.